include/mega/filesystem.h

```diff
@@ -160,11 +160,16 @@ class MEGA_API LocalPath
     void trimNonDriveTrailingSeparator(const FileSystemAccess& fsaccess);
     bool findNextSeparator(size_t& separatorBytePos, const FileSystemAccess& fsaccess) const;
     bool findPrevSeparator(size_t& separatorBytePos, const FileSystemAccess& fsaccess) const;
+    bool endsInSeparator(const FileSystemAccess& fsaccess) const;
+
+    // get the index of the leaf name.  A trailing / is considered part of the leaf.
```

Comment:
```suggestion
    // get the index of the leaf name.  A trailing separator is considered part of the leaf.
```
Not worth a commit ;-)